compiler+runtime/test/jank/form/case/pass-statement-position.jank

```diff
@@ -0,0 +1,9 @@
+(let [xfn (fn []
+            (case 3/4
+              4/3 :four-thirds
+              3/4 :three-fourths
```

Comment:
Let's be certain this runs by putting a `(def ret :success)` for the matched branch. Then return `ret` at the end of the file.